docs/PRIORITIES.md

```diff
@@ -1,46 +1,35 @@
 # Post-0.25/GoS Pre-Release
 
-## Staking/Slashing/Stability
-
-- Other slashing issues blocking for launch - [#1256](https://github.com/cosmos/cosmos-sdk/issues/1256)
-- Miscellaneous minor staking issues
-  - [List here](https://github.com/cosmos/cosmos-sdk/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Astaking+label%3Aprelaunch)
-  - Need to figure out scope of work here to estimate time
-  - @rigelrozanski to start next
-- Consider "tombstone" / "prison" - double-sign and you can never validate again - https://github.com/cosmos/cosmos-sdk/issues/2363
```

Comment:
Have we decided to reject this idea?

Reply:
I think this got decisively put out of scope for launch. I don't think the idea has been rejected.

Reply:
Hmm the last note on the issue is `if-have-time-prelaunch`, I think from an SDK meeting?

"Tombstone" could simplify parts of the state machine substantially, it removes the need for a slashing period.

Was there another discussion I missed or another issue?